### ISSUE TITLE
Links in gophermap not displayed

### ISSUE THREAD

**Issue by korwisi** [skyjake/lagrange#230]
Either, my `gophermap` is malformed or there is an issue with links in `gophermap` in the desktop version of Lagrange.

Tested `gophermap` ([gopher://sdf.org/1/users/kristof/linktest](gopher://sdf.org/1/users/kristof/linktest)):
```
Link test

HTTP
hAn http link   URL:http://kristof.sdf.org

HTTPS
hAn https link  URL:https://sdf.org

Gopher
hA gopher link  URL:gopher://sdf.org

Gemini
hA gemini link  URL:gemini://sdf.org
```

Rendered result (Lagrange v1.2.3, macOS 11.2.3):
<img width="960" alt="lagrange-1.2.3-macos-linktest" src="https://user-images.githubusercontent.com/928900/112765028-8a0bc800-900b-11eb-979d-fe163d4c17df.png">

Lagrange (1.3.0) for iOS renders it correctly:
![lagrange-1.3.0-ios-linktest](https://user-images.githubusercontent.com/928900/112765091-cfc89080-900b-11eb-9977-5e68b90c71df.png)

Interestingly, links on [gopher://sdf.org](gopher://sdf.org/) are rendered but I don't have access to the corresponding `gophermap`:
<img width="960" alt="lagrange-1.2.3-macos-sdforg" src="https://user-images.githubusercontent.com/928900/112765204-57ae9a80-900c-11eb-9bf8-c706f6111076.png">

**Comment by korwisi** [comment]
If I understood it correctly, it is already addressed in the next release with https://github.com/skyjake/lagrange/commit/5d3567bd26ad1031e4ec08ccd2c36f5f1f646ca7. My bad! Feel free to close the issue, if this is the case :-)

**Comment by stonedbovines** [comment]
Just to add my 2p-worth, the links don't currently display on the Win 10 desktop either.

**Comment by skyjake** [comment]
Thanks for reporting! This has indeed been fixed for v1.3. The previous report is #208 (although the title is about git links, it's the same Gopher line type).
